### PR TITLE
Upgraded most dependencies and IMS org.

### PR DESCRIPTION
**Pull request by Aaronius**
## Description
I upgraded my computer and with that I installed the latest versions of node and npm. When I ran `npm install` or `npm ci` on this repository, I got several errors that seemed to be due to incompatibilities between the dependencies and my version of node or npm.

This PR updates most dependencies, which allows me to successfully install all dependencies on my machine. Some of the dependencies I did not update either (1) caused tests to fail or (2) were going to cause conflict headaches with the open pull requests on this repository (prettier and eslint). After we release and have no open pull requests, it would probably be a good idea to upgrade the remaining dependencies.

While I was doing this, the UnifiedEdgeQEOnly org was deprovisioned from Adobe Experience Platform, so I also made the necessary changes to transition to the UnifiedJS QE Only org. I also re-created the necessary schema that we had lost.

## Motivation and Context
Be able to install dependencies. Use a properly provisioned org.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I've updated the schema in extension.json or no changes are necessary.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "plugins": [
-    "@babel/plugin-proposal-class-properties",
```
Comment (Aaronius) [inline .babelrc:3]: This is apparently built in now.

File: package.json [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
     "prepush-msg": "echo 'Running pre-push scripts...' && exit 0",
     "precommit-msg": "echo 'Running pre-commit scripts...' && exit 0",
+    "postinstall": "patch-package",
     "sandbox": "reactor-sandbox",
     "build": "./scripts/build.js",
     "build:prod": "NODE_ENV=production npm run build",
     "package": "npm run build:prod && npx @adobe/reactor-packager",
     "dev": "./scripts/dev.js",
-    "lint": "eslint \"*.js\" \"src/**/*.{js,jsx}\" \"test/**/*.{js,jsx}\" \"scripts/**/*.js\"",
+    "lint": "eslint \"*.js\" \"src/**/*.{js,jsx}\" \"test/**/*.{js,jsx}\" \"scripts/**/*.js\" --fix",
```
Comment (Aaronius) [inline package.json:16]: I figured we should try to automatically try to fix lint problems if we can.

File: test/functional/dataElements/identityMap/identityMap.spec.js [hunk 3]
```diff
@@ -23,16 +24,9 @@ const identityMapViewController = createExtensionViewController(
...
   const accessToken = await getAdobeIOAccessToken();
   const initInfo = {
-    extensionSettings: {
-      instances: [
-        {
-          name: "alloy1",
-          edgeConfigId: "74580452-647b-4797-99af-6d0e042435ec"
-        }
-      ]
-    },
```
Comment (Aaronius) [inline test/functional/dataElements/identityMap/identityMap.spec.js:33]: What we had here just wasn't necessary and ended up being more to maintain.

**Review by jfkhoury — APPROVED** [review]
👍

**Review by ninaceban — APPROVED** [review]
looks good to me 👍

**Pull request merged**